pokebattle/urls.py

```diff
@@ -5,14 +5,21 @@
 
 import django_js_reverse.views
 
+from battles.views import PokemonListAPIView
+
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls, name='admin'),
     url(r'^jsreverse/$', django_js_reverse.views.urls_js, name='js_reverse'),
-    # App
+
+    # Pokebattle
+
     url(r'^', include('users.urls', namespace='auth')),
     url(r'^$', TemplateView.as_view(template_name='common/index.html'), name='home'),
-    url(r'^battles/', include('battles.urls', namespace='battles'))
+    url(r'^battles/', include('battles.urls', namespace='battles')),
+
+    # api
+    url(r'^api/pokemon$', PokemonListAPIView.as_view(), name='api-pokemon-list'),
```

Comment:
I think you forgot the trailing bar in the url here

Reply:
Done.